frontend-shared/styles/mixins/focus.scss

```diff
@@ -1,3 +1,26 @@
+@use '../variables' as var;
+
+/**
+ * Add stylized focus to interactive elements such <input> or <textarea>
+ *
+ * @param {boolean} $inset - Does the outline render inset or not
```

Comment:
Can this comment be expanded to explain why it matters if an outline is `inset` or not? What information would a user need to make a decision one way or the other?